sem-install

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+VERSION="0.3"
+SUPPORTED_LANGUAGES=(php ruby)
+# Misc
+GO_URL="https://storage.googleapis.com/golang"
+DATE_FORMAT='%H:%M %d/%m/%Y'
+HOST="packages.semaphoreci.com"
+
+install::check_precompiled() {
+  response=$(curl --write-out %{http_code} --head --silent --output /dev/null http://$HOST/${language}/${language_version}.tar.gz)
+  echo $response
+}
```

Comment:
This function should have boolean response. So the rest of the system doesn't need to know underlaying implementation. e.g. We are kind of leaking `200` HTTP response out of this function.

Reply:
This function could be called `is_available`

Reply:
The curl command can timeout (or brake in unforseen ways).

I propose a stronger check with:

```
exit_code == 0 && status == 200
```

(pseudocode)

---

Also, let's define a timeout value on the curl call explicitly (I am not sure what is the default, 1 minute?).

Reply:
Done